### PR TITLE
WIP DO NOT REVIEW Batchnorm packed

### PR DESCRIPTION
**Pull request by annxingyuan**
#### Description

---

##### For repository owners only:

Please remember to apply all applicable tags to your pull request.
Tags: FEATURE, BREAKING, BUG, PERF, DEV, DOC, SECURITY

For more info see: https://github.com/tensorflow/tfjs/blob/master/DEVELOPMENT.md

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-core/1326)

### CODE REVIEW

**Comment by nsthorat** [comment]
FYI -- I usually delete and create a new dev branch every time I merge a PR, it helps keep the git commit history for a PR sane :). Up to you though.

**Comment by annxingyuan** [comment]
@nsthorat what do you mean exactly? sounds like something I should do! (also this PR won't be merged - I'm creating smaller PR's out of it - maybe I should close it to avoid confusion?)

**Comment by annxingyuan** [comment]
@nsthorat oh ok - yeah I think I've been doing that - please let me know if you see otherwise though! the commits in the PR are actually from the same dev branch lol...

**Comment by nsthorat** [comment]
Oh got it, sorry :)
